lib/src/navbar_router.dart

```diff
@@ -152,6 +158,23 @@ class NavbarRouter extends StatefulWidget {
   /// Set to true will hide the badges when the tap on the navbar icon.
   final bool hideBadgeOnPageChanged;
 
+  /// Set to true will opt-in to horizontally swipeable navigation bar.
+  final bool swipeable;
+
+  /// Configure the swipeable area on the left edge of the screen
+  ///
+  /// By default, it will be: center aligned, width = 50 pixels, top = 50 pixels, height = 0.8 x screen height
+  ///
+  /// Only take effect if **[swipeable]** is set to true
+  final Rect? swipeableLeftArea;
```

Comment:
and where would you place the swipable area if it controls both direction? in the middle? on the bottom bar? tell me another more intuitive position for swiping left and right than left and right themselves...

Reply:
Isn;t that question for the client to decide? We are just exposing the swipeableArea and the client can specify `Rect.LTWH`, We should place it in the host application (package) so that the client can occupy maximum space available above the bottom navbar.